### ISSUE TITLE
Android problem with function: replaceExistingNonRootView & removeSubviewsFromContainerWithID (RN 0.74.0-rc.3)

### ISSUE THREAD

**Issue by 404-html** [software-mansion/react-native-reanimated#5795]
### Description

When running `react-native run-android` the build fails with following errors:

```
..../node_modules/react-native-reanimated/android/src/reactNativeVersionPatch/ReanimatedUIManager/latest/com/swmansion/reanimated/layoutReanimation/ReanimatedUIManager.java:103:
error: cannot find symbol super.replaceExistingNonRootView(oldTag, newTag); ^ symbol: method replaceExistingNonRootView(int,int)

.../node_modules/react-native-reanimated/android/src/reactNativeVersionPatch/ReanimatedUIManager/latest/com/swmansion/reanimated/layoutReanimation/ReanimatedUIManager.java:116:
error: cannot find symbol super.removeSubviewsFromContainerWithID(containerTag); ^ symbol: method removeSubviewsFromContainerWithID(int)
```

Looks like a problem with some three years old code.
I tried: clearing yarn cache, re-installing node modules, gradlew clean - still the same problem.

### Steps to reproduce

1. Create new app targeting RN 0.74.0-rc.3
2. Add reanimated to the project
3. Run `react-native run-android`

### Snack or a link to a repository

n/a - build problem

### Reanimated version

3.8.0

### React Native version

0.74.0-rc.3

### Platforms

Android

### JavaScript runtime

Hermes

### Workflow

React Native

### Architecture

None

### Build type

Debug app & dev bundle

### Device

Android emulator

### Device model

Pixel 8

### Acknowledgements

Yes

**Comment by 404-html** [comment]
After commenting out problematic functions in `ReanimatedUIManager.java` the problem is gone.

**Comment by Latropos** [comment]
Hi @404-html Unafortunatelly, as you can read in our documentation: https://docs.swmansion.com/react-native-reanimated/docs/guides/compatibility we don't support react-native 0.74 yet. Please try 0.73

**Comment by migueldaipre** [comment]
Maybe fixed by #5815

**Comment by Latropos** [comment]
Yes, good catch @migueldaipre!
